src/ucp/api/ucp.h

```diff
@@ -13,6 +13,10 @@
 #include <ucs/config/types.h>
 #include <ucs/sys/math.h>
 #include <stdio.h>
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
```

Comment:
this is dangerous thing in public API, we should define a structure which is platform-independent
@shamisp